dependencies/pom.xml

```diff
@@ -134,7 +134,6 @@
               <mainClass>com.google.cloud.tools.opensource.dependencies.Java8IncompatibleReferenceCheck</mainClass>
```

Comment:
I'll remove this class once protobuf-java fixes the issue.
https://github.com/protocolbuffers/protobuf/issues/7827